### DISCUSSION TITLE
Pyvista Won't Render Output Properly

### DISCUSSION THREAD

**Discussion by arotker45** [dpploy/engy-5310#31] [General]
I'm debugging the custom boundary condition and the units I selected for the solid phase right now. I managed to get my notebook to visualize the .vtu file with a workaround since WSL doesn't support frame buffer, but it just shows a grey rectangle even though I checked the output file and it has data in it.
![thing thing](https://user-images.githubusercontent.com/46360914/116341858-ab4d0900-a7af-11eb-87f0-1aaa766b9909.png)

I can't link the code directly because its in a notebook.

**Comment by dealmeidavf** [comment]
This is what I see from running your report:

![snapshot](https://user-images.githubusercontent.com/378189/116342499-535ed400-a7a8-11eb-81f3-52f7492e014e.png)

We need to truncate the domain on the `pyvista` script to show a portion of the domain.

I do not know why your notebook is not showing the domain. How about a different browser? Windows Edge? or can you run a browser from the command line `Ubuntu`?

**Reply by arotker45** [reply]
It doesn't render in edge either. It looks correct from what I'm seeing at least; The temperature is high because the flux on the interior edge has just been held close to zero.

**Reply by dealmeidavf** [reply]
We need an axial and radial plot. The colorful map would not give enough info. Let's add plots from the `csv` data; just like in the course lecture notebooks.

**Comment by AnthonyB08** [comment]
If you are using jupyter notebook, add a cell above (before importing pyvista) and input pip install pyvista
